com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -1941,7 +1941,7 @@ internal void EvaluateShadowState(HDCamera hdCamera, in ProcessedLightData proce
                 bool validShadow = false;
                 if (processedLight.gpuLightType == GPULightType.Point
                     || processedLight.gpuLightType == GPULightType.Rectangle
-                    || (processedLight.gpuLightType == GPULightType.Spot && processedLight.lightVolumeType == LightVolumeType.Cone))
+                    || processedLight.gpuLightType == GPULightType.Spot)
```

Comment:
why is this condition remove? guess it is required for screen space shadow? (poking @JulienIgnace-Unity  just in case)

Reply:
LightVolume was the "thing" Morten added for clustered lighting and I removed it.

It is possible that I've made a mistake modifying this particular part of code.